src/Workspaces/Remote/ServiceHub/Services/DiagnosticAnalyzer/DiagnosticComputer.cs

```diff
@@ -1,493 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
This entire *complex* type was never used in product scenarios.  But was only called from tests... yikes.